bitnami/prometheus-operator/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.40.0
 description: The Prometheus Operator for Kubernetes provides easy monitoring definitions for Kubernetes services and deployment and management of Prometheus instances.
 name: prometheus-operator
-version: 0.22.3
+version: 0.22.4
```

Comment:
I think this should be a minor bump

Reply:
@dani8art if we follow your approach then we will need a major bump as the change is breaking backwards compatibility and users need to perform manual steps to migrate no?